pkg/services/authz/rbac.go

```diff
@@ -159,13 +161,24 @@ func newRemoteRBACClient(clientCfg *authzClientSettings, tracer trace.Tracer) (a
 		}
 	}
 
-	conn, err := grpc.NewClient(
-		clientCfg.remoteAddress,
+	authzRequestDuration := promauto.With(reg).NewHistogramVec(prometheus.HistogramOpts{
+		Name:    "authz_server_client_request_duration_seconds",
+		Help:    "Time spent executing requests to authz server.",
+		Buckets: prometheus.ExponentialBuckets(0.008, 4, 7),
```

Comment:
I'd suggest to use native histograms for new code:

```suggestion
			NativeHistogramBucketFactor:     1.1,
			NativeHistogramMaxBucketNumber:  160,
			NativeHistogramMinResetDuration: time.Hour,
```